[p5.js 2.0 Bug Report]: New vertex functions missing in p5.Graphics objects

### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [x] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### p5.js version

2.0.0

### Web browser and version

Firefox 137.0.2 / archlinux

### Operating system

Arch Linux 2024-04-18

### Steps to reproduce this

### Steps:
1. Create a p5.Graphics object
2. Try using one of the new vertex functions, e.g. `splineVertex` on it.
3. Get an error about that function missing.
4. (workaround) Call that function anyway using `._renderer.splineVertex`

### Snippet:

[Reproduction project](https://editor.p5js.org/bojidar-bg/sketches/0w4UgLOgJ)

```js
function setup() {
  createCanvas(100, 100)
  let g = createGraphics(width, height)
  g.beginShape()
  g.splineVertex(10, 10)
  g.splineVertex(20, 10)
  g.splineVertex(20, 20)
  g.endShape()
  image(g, 0, 0) // #7579
}
```

Hey @ksen0, I’ve added p5.Graphics.prototype.splineVertex in my PR to address the missing method error — it now proxies to this._renderer.splineVertex(...). After adding it, the test passes successfully (AssertionError: expected undefined to be truthy is gone).
However, I’m unsure if the placement of this extension is ideal. Currently, I’ve placed it src/core/p5.Graphics.js, but it might make more sense to move it somewhere more appropriate in the codebase.
Would love your guidance on where this kind of prototype patch is best placed structurally. Or my pr need Changes

--- a/src/core/p5.Graphics.js
+++ b/src/core/p5.Graphics.js
@@ -23,6 +23,7 @@ import material from '../webgl/material';
 import creatingReading from '../color/creating_reading';
 import trigonometry from '../math/trigonometry';
 import { renderers } from './rendering';
+import customShapes from '../shape/custom_shapes';
 
 class Graphics {
   constructor(w, h, renderer, pInst, canvas) {
@@ -677,6 +678,7 @@ function graphics(p5, fn){
   attributes(p5, p5.Graphics.prototype);
   curves(p5, p5.Graphics.prototype);
   vertex(p5, p5.Graphics.prototype);
+  customShapes(p5, p5.Graphics.prototype);
 
   setting(p5, p5.Graphics.prototype);
   loadingDisplaying(p5, p5.Graphics.prototype);

--- a/test/unit/core/p5.Graphics.js
+++ b/test/unit/core/p5.Graphics.js
@@ -117,6 +117,14 @@ suite('Graphics', function() {
     });
   });
 
+  suite('p5.Graphics.prototype.splineVertex', function() {
+    test('should be a function in graphics', function() {
+      let g = myp5.createGraphics(10, 10);
+      assert.ok(g.splineVertex);
+      assert.typeOf(g.splineVertex, 'function');
+    });
+  });
+  
   suite('p5.Graphics.resizeCanvas', function() {
     let glStub;
 

